### ISSUE TITLE
Enhancement Request: Dynamic Episode Link Generation from Official API

### ISSUE THREAD

**Issue by Zai-Kun** [FireHead90544/SenPY#10]
You're creating episode links statically, which could lead to issues. There's a way to fetch the episode links from the official API, like this:

Here's the API link: [https://ajax.gogo-load.com/ajax/load-list-episode?ep_start=0&ep_end={EP_END_HERE}&id={ANIME_ID_HERE}](https://ajax.gogo-load.com/ajax/load-list-episode?ep_start=0&ep_end=%7BEP_END_HERE%7D&id=%7BANIME_ID_HERE%7D)

You can obtain the anime ID like this:

```python

anime_id = soup.find("input", {"id": "movie_id"})['value']
```
You can extract the anime ID from the anime's URL (e.g., https://gogoanimehd.io/category/suzume-no-tojimari-dub).

You can obtain the last episode number like this (copied this from your original code):

```python
last = int([i for i in soup.select("#episode_page")[0]][-2].a['ep_end'])
```

**Comment by FireHead90544** [comment]
Awesome, thanks for the info. I'd check it out.

**Comment by Zai-Kun** [comment]
> Awesome, thanks for the info. I'd check it out.

Sure, no problem.

**Comment by FireHead90544** [comment]
Thanks for the info buddy! Just updated the logic and released [v1.1.0](https://github.com/FireHead90544/SenPY/releases/tag/v1.1.0).
